README.md

```diff
@@ -6,10 +6,10 @@ This action executes `yamllint` (https://github.com/adrienverge/yamllint) agains
 
 ### Required parameters
```

Comment:
This makes it even neater workflow without the need to have any parameters in `with:`. Do you mind moving the param `file_or_dir` to Optional parameters as it is not required anymore?

Reply:
Good point...
Changed...